Mathlib/Data/Option/Basic.lean

```diff
@@ -362,6 +362,25 @@ theorem casesOn'_none_coe (f : Option α → β) (o : Option α) :
     casesOn' o (f none) (f ∘ (fun a ↦ ↑a)) = f o := by cases o <;> rfl
 #align option.cases_on'_none_coe Option.casesOn'_none_coe
 
+
+section recursor_workarounds
```

Comment:
Theres a lean4 issue that we should reference in a comment here. Can you look for other `rec_eq_recC` lemmas and copy the comment we used before?